app/src/main/java/com/github/epfl/meili/profile/favoritepois/FavoritePoisRecyclerAdapter.kt

```diff
@@ -0,0 +1,40 @@
+package com.github.epfl.meili.profile.favoritepois
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.github.epfl.meili.R
+import com.github.epfl.meili.models.FavoritePointOfInterest
+import com.github.epfl.meili.util.MeiliRecyclerAdapter
+import java.text.SimpleDateFormat
+import java.util.*
+
+class FavoritePoisRecyclerAdapter : MeiliRecyclerAdapter<FavoritePointOfInterest>() {
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerView.ViewHolder {
+        return PoiViewHolder(
+            LayoutInflater.from(parent.context).inflate(R.layout.poi, parent, false)
+        )
+    }
+
+    override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) {
+        val pvh = (holder as PoiViewHolder)
+        pvh.bind(items[position])
+    }
+
+
+    class PoiViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+        private val poiName: TextView = itemView.findViewById(R.id.poi_name)
+        private val visitedDate: TextView = itemView.findViewById(R.id.visited_date)
```

Comment:
should we have a visited date for favourite pois? The user could potentially favourite a POI that they haven't visited.

Reply:
Yes i forgot to change this, I converted it to a 'favourited' date. To me it seems like it could be useful to have that.

Reply:
What could the favourite date be useful for?